presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -328,8 +329,7 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
         builder.add(new IterativeOptimizer(
                 stats,
                 ImmutableList.of(new PartialAggregationPushDown(metadata.getFunctionRegistry())),
-                ImmutableSet.of(new PushPartialAggregationThroughJoin())));
-        builder.add(new PartialAggregationPushDown(metadata.getFunctionRegistry()));
+                ImmutableSet.of(new PushPartialAggregationThroughJoin(), new PushPartialAggregationThroughExchange(metadata.getFunctionRegistry()))));
```

Comment:
* deprecate the old one
* glue this iterative optimizer with the subsequent optimizer